src/s3/headobject.cpp

```diff
@@ -14,6 +14,8 @@ namespace asio = boost::asio;
 namespace beast = boost::beast;
 namespace fs = irods::experimental::filesystem;
 
+static const std::string date_format{"{:%a, %d %b %Y %H:%M:%S %Z}"};
```

Comment:
So the date format changes depending on the S3 API call?

Reply:
From testing with aws command line tool using the --debug flag, that is what I got.  The aws command line tool seems to like it and converts the dates correctly to its output format for each different API.